t/manifest.t

```diff
@@ -0,0 +1,15 @@
+#!perl -T
+use 5.006;
+use strict;
+use warnings;
+use Test::More;
+
+unless ($ENV{RELEASE_TESTING}) {
+    plan(skip_all => "Author tests not required for installation");
+}
+
+my $min_tcm = 0.9;
+eval "use Test::CheckManifest $min_tcm";
+plan skip_all => "Test::CheckManifest $min_tcm required" if $@;
```

Comment:
please allow all tests on travis. I don't trust anybody to run any special tests before release. This also applies to the tests below.